services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/cli.py

```diff
@@ -72,6 +58,3 @@ def evaluate(ctx: typer.Context, user_id: int) -> None:
     )
     if not settings.RESOURCE_USAGE_TRACKER_PROMETHEUS:
         raise ConfigurationError(msg="no valid prometheus endpoint defined!")
```

Comment:
so what is the point of this command? I don't get why you keep it there.

Reply:
you are right it is not needed, I have not yet focused on `cli` so it is some old code i will remove it